datadog/prerun.sh

```diff
@@ -8,4 +8,9 @@ if [ "$DYNOTYPE" == "run" ]; then
   # more info here: https://docs.honeycomb.io/getting-data-in/ruby/beeline/#using-env-variables-to-control-framework-integrations
   export HONEYCOMB_DISABLE_AUTOCONFIGURE="true"
   export HONEYCOMB_INTEGRATIONS=rails,rake,active_support
+
+  # Set app version based on HEROKU_SLUG_COMMIT
+  if [ -n "$HEROKU_SLUG_COMMIT" ]; then
+    DD_VERSION=$HEROKU_SLUG_COMMIT
```

Comment:
Should this be `export`ed like the ones above?

```suggestion
    export DD_VERSION=$HEROKU_SLUG_COMMIT
```

Reply:
I think DD's env vars do not require to be exported. `DISABLE_DATADOG_AGENT` wasn't exported in this file.

Reply:
Oh interesting. Well, let's give it a shot and see how it goes!